app/src/test/java/com/willowtree/vocable/FakeCategoriesUseCase.kt

```diff
@@ -23,7 +24,15 @@ class FakeCategoriesUseCase : ICategoriesUseCase {
     )
 
     override fun categories(): Flow<List<Category>> {
-        return _categories
+        return _categories.map {
```

Comment:
I had to make this change to get the fake to match the behavior of the real use case. (I know, I know. The real solution is to refactor the test to use the real use case. I am just not prepared to do that as a part of this PR.)